package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "auth_service_vi",
+  "version": "1.0.0",
+  "description": "The authentication service for Edustripe.com",
+  "main": "index.js",
+  "scripts": {
+    "build": " rm -rf dist && babel index.js -d dist && babel app -d dist/app",
+    "db:migrate": "sequelize db:migrate",
+    "db:migrate:undo": "sequelize db:migrate:undo:all",
+    "db:refresh": "npm run db:migrate:undo && npm run db:migrate && npm run db:seed",
+    "db:seed": "sequelize db:seed:all",
+    "start": "node dist/index.js",
+    "lint": "eslint . --fix",
+    "dev": "cross-env NODE_ENV=development nodemon --exec babel-node index.js",
+    "test": "cross-env NODE_ENV=test npm run db:refresh && cross-env NODE_ENV=test nyc mocha --require @babel/register --exit"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/edustripetech/auth_service_v1.git"
+  },
+  "author": "Edustripe Team",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/edustripetech/auth_service_v1/issues"
+  },
+  "homepage": "https://github.com/edustripetech/auth_service_v1#readme",
+  "dependencies": {
+    "axios": "^0.19.2",
```

Comment:
Why do we have axios as one of our dependencies? I don't see it used anywhere.

Reply:
It will be used  for communication between services. Same way sequelize have  not  yet been used but we know we'll  use it eventually so I believe  it should  be part  of the setup.

Reply:
All right then. Great Job.